### PR TITLE
Add xml-doc comments to RepositoryReadConductor. Fixes issue #11

### PR DESCRIPTION
**Pull request by vmrocha**
Adds xml-doc comments to the RepositoryReadConductor class public members.

- [ ] Related GitHub issue(s) linked in PR description
- [ ] Destination branch merged, built and tested with your changes
- [ ] Code formatted and follows best practices and patterns
- [ ] Code builds cleanly (no _additional_ warnings or errors)
- [ ] Manually tested
- [ ] Automated tests are passing
- [ ] No _decreases_ in automated test coverage
- [ ] Documentation updated (readme, docs, comments, etc.)
- [ ] Localization: No hard-coded error messages in code files (_minimally_ in string constants)

### CODE REVIEW

File: src/AndcultureCode.CSharp.Conductors/RepositoryReadConductor.cs [hunk 1]
```diff
@@ -27,6 +34,10 @@ public class RepositoryReadConductor<T> : Conductor, IRepositoryReadConductor<T>
 
         #region Constructor
 
+        /// <summary>
+        /// Creates and instance of RepositoryReadConductor for an <see cref="IRepository{T}"/> instance.
```
Comment (brandongregoryscott) [inline src/AndcultureCode.CSharp.Conductors/RepositoryReadConductor.cs:38]: Typo - `and` vs `an`

File: src/AndcultureCode.CSharp.Conductors/RepositoryReadConductor.cs [hunk 2]
```diff
@@ -39,6 +50,17 @@ public RepositoryReadConductor(IRepository<T> repository)
...
 
+        /// <summary>
+        /// Find all filtered, sorted and paged.
+        /// </summary>
+        /// <param name="filter">Filter to be used for querying.</param>
+        /// <param name="orderBy">Properties that should be used for sorting.</param>
+        /// <param name="includeProperties">Navigation properties that should be included.</param>
+        /// <param name="skip">Number of entities that should be skipped.</param>
+        /// <param name="take">Number of entities per page.</param>
+        /// <param name="ignoreQueryFilters">If previous applied filters should be ignore.</param>
```
Comment (brandongregoryscott) [inline src/AndcultureCode.CSharp.Conductors/RepositoryReadConductor.cs:61]: Thanks for filling these in! While this is close, this parameter actually applies to the concept of [EF Core Global Query Filters](https://docs.microsoft.com/en-us/ef/core/querying/filters). This comment would better read something along the lines of "If true, global query filters will be ignored for this query." The same would apply for any other methods in this conductor that have the same param.

**Review by brandongregoryscott — CHANGES_REQUESTED** [review]
Thanks for the contribution, @vmrocha! Left some feedback for some revisions to the descriptions. Also, please be mindful of the pull request checklist and fill out any steps that apply to the given issue. This is just a documentation update, so most of them probably _won't_ apply, but please at least link the related issue in the description with [closing keywords](https://docs.github.com/en/enterprise/2.16/user/github/managing-your-work-on-github/closing-issues-using-keywords#closing-an-issue-in-the-same-repository)

File: src/AndcultureCode.CSharp.Conductors/RepositoryReadConductor.cs (hunk 1 above)
Comment (vmrocha) [reply src/AndcultureCode.CSharp.Conductors/RepositoryReadConductor.cs:38]: Good catch!

File: src/AndcultureCode.CSharp.Conductors/RepositoryReadConductor.cs (hunk 2 above)
Comment (vmrocha) [reply src/AndcultureCode.CSharp.Conductors/RepositoryReadConductor.cs:61]: Thank you for explaining that. I didn't know!

**Review by brandongregoryscott — APPROVED** [review]
Thanks for making those updates, @vmrocha!

**Pull request merged**